manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -59,125 +59,182 @@ spec:
           - -c
           - |
             set -ex
-
-            PROXY_ORG_SRC_MARK=0x4d2/0xfff
+  {{- if .Values.ztunnel.ebpfRedirect }}
```

Comment:
Could we consider actually making this entire rule block a templated value, rather than a set of hardcoded commands gated by a single conditional?

That feels _much_ less confusing than trying to hardcode a "golden" set of nftables/iptables rules, tightly-coupled to $whatever_bpf_prog, that is sufficiently pluggable with env vars.


If someone wants to bring their own eBPF progs for redirection, they should be able to do that without directly hacking the daemonset template.

Reply:
+1 to that ^^

Reply:
This is on todo list, we need to decide where to put this flag, one possible way is to make it part of meshconfig.

Reply:
The flag can go in meshConfig or not - but just to be clear I'm suggesting that _all_ the iptables/nftables rules here be moved out of this daemonset file and into a `values.yaml` property.

So lines 73-117  and lines 119-236